dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitLabSourceCodeRepo.java

```diff
@@ -280,11 +279,6 @@ public SourceFile getSourceFile(String path, String id, String branch, Descripto
         return null;
     }
 
-    @Override
```

Comment:
Will deleting this and the bitbucket counterpart cause app logs to show up for these two source control methods?

Reply:
The methods weren't and aren't invoked for non-GitHub source controls: [LambdaEventResource](https://github.com/dockstore/dockstore/blob/4e437e6e3bdaba164842c87a49283496bfaa87d9/dockstore-webservice/src/main/java/io/dockstore/webservice/resources/LambdaEventResource.java#L67). And the code does an [explicit cast](https://github.com/dockstore/dockstore/blob/4e437e6e3bdaba164842c87a49283496bfaa87d9/dockstore-webservice/src/main/java/io/dockstore/webservice/resources/LambdaEventResource.java#L86) to GitHubSourceRepo.

I think it's OK, but I'll ponder some more.

Reply:
* `io.dockstore.webservice.helpers.SourceCodeRepoInterface#getOrganizations` was previously only called from `io.dockstore.webservice.resources.UserResource#getMyGitHubOrgs`, and only if there was a GitHub token. As part of this PR I have removed `getMyGitHubOrgs`.
* `io.dockstore.webservice.helpers.SourceCodeRepoInterface#getOrganizations` is now more generic, and has a default implementation.

TLDR; I feel this is good.